README.md

```diff
@@ -417,6 +417,45 @@ Other Style Guides
 
 **[⬆ back to top](#table-of-contents)**
 
+<a name="arrays--bracket-newline"></a><a name="4.5"></a>
```

Comment:
Numbered links are legacy, and should never be added or changed - let's remove the one on line 420 (but keep the visual number on line 421)